src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -2491,7 +2491,10 @@ ITypeSymbol ITypeSymbolInternal.GetITypeSymbol()
         internal abstract bool IsExtension { get; }
         internal abstract bool IsExplicitExtension { get; }
 
-        internal abstract TypeSymbol? ExtendedTypeNoUseSiteDiagnostics { get; }
+        internal TypeSymbol? ExtendedTypeNoUseSiteDiagnostics
```

Comment:
>ExtendedTypeNoUseSiteDiagnostics [](http://example.com/codeflow?start=29&length=32)

It feels like presence of this API makes it easy to fail to propagate `basesBeingResolved`.

Reply:
I didn't want to cause too much churn (many references in test file). But I think you're right. Removed the property in commit 5.